src/python/qequlacs/utils.py

```diff
@@ -38,24 +73,45 @@ def append_gate(gate, qulacs_circuit):
     elif len(gate.qubits) == 2:
         index_1 = gate.qubits[0].index
         index_2 = gate.qubits[1].index
+
         if gate.name == 'CNOT':
             qulacs_circuit.add_CNOT_gate(index_1, index_2)
         elif gate.name == 'SWAP':
             qulacs_circuit.add_SWAP_gate(index_1, index_2)
         elif gate.name == 'XX':
-            qulacs_circuit.add_multi_Pauli_rotation_gate([index_1, index_2], [1,1], -gate.params[0]*2/np.pi)
+            qulacs_circuit.add_multi_Pauli_rotation_gate([index_1, index_2], [1,1], -angle)
         elif gate.name == 'YY':
-            qulacs_circuit.add_multi_Pauli_rotation_gate([index_1, index_2], [2,2], -gate.params[0]*2/np.pi)
+            qulacs_circuit.add_multi_Pauli_rotation_gate([index_1, index_2], [2,2], -angle)
         elif gate.name == 'ZZ':
-            qulacs_circuit.add_multi_Pauli_rotation_gate([index_1, index_2], [3,3], -gate.params[0]*2/np.pi)
+            qulacs_circuit.add_multi_Pauli_rotation_gate([index_1, index_2], [3,3], -angle)
```

Comment:
Can someone explain why we're dropping the factor of `2/np.pi`?

Reply:
Different frameworks use gate parameters with different conventions – with/without pi, etc.
And @vsergeyev figured out that these multipliers here are a correct way to do that.

Since I had issues with these before, I wouldn't merge it until it gives consistent results with other frameworks. If it doesn't then we'll fix it.

Reply:
@simonwa7 this is from the assumption we have 
`exponent=gate.params[0]/np.pi`
in https://github.com/zapatacomputing/z-quantum-core/blob/master/src/python/zquantum/core/circuit/_circuit.py#L757

and in qulacs_simulator it is:
`qulacs_circuit.add_multi_Pauli_rotation_gate(indices, [1, 1], -np.pi * op.gate._exponent)`

so this result in:

`qulacs_circuit.add_multi_Pauli_rotation_gate(indices, [1, 1], -np.pi * gate.params[0] / np.pi)`